Mage.Tests/src/test/java/org/mage/test/cards/watchers/IanMalcolmChaoticianTests.java

```diff
@@ -0,0 +1,108 @@
+package org.mage.test.cards.watchers;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestCommander4Players;
+
+/**
+ *
+ * @author jimga150
+ */
+public class IanMalcolmChaoticianTests extends CardTestCommander4Players {
+
+    @Test
+    public void testManaCostsandWatcher() {
+
+        skipInitShuffling();
+
+        // Whenever a player draws their second card each turn, that player exiles the top card of their library.
+        // During each player's turn, that player may cast a spell from among the cards they don't own exiled with
+        // Ian Malcolm, Chaotician, and mana of any type can be spent to cast it.
+        addCard(Zone.BATTLEFIELD, playerA, "Ian Malcolm, Chaotician");
+
+        // Flying
+        // At the beginning of your draw step, draw an additional card.
+        // At the beginning of your end step, discard your hand.
+        addCard(Zone.BATTLEFIELD, playerA, "Avaricious Dragon");
+        addCard(Zone.BATTLEFIELD, playerB, "Avaricious Dragon");
+        addCard(Zone.BATTLEFIELD, playerC, "Avaricious Dragon");
+        addCard(Zone.BATTLEFIELD, playerD, "Avaricious Dragon");
+
+        addCard(Zone.LIBRARY, playerA, "Horde of Notions", 3);
+        addCard(Zone.LIBRARY, playerB, "Fusion Elemental", 10);
+        addCard(Zone.LIBRARY, playerC, "Chromanticore", 10);
+        addCard(Zone.LIBRARY, playerD, "Garth One-Eye", 10);
+
+        addCard(Zone.BATTLEFIELD, playerA, "Plains", 15);
+
+        // Should be able to cast all cards exiled from other players' decks
+        checkPlayableAbility("Can't cast card exiled with this Ian Malcolm", 5, PhaseStep.PRECOMBAT_MAIN, playerA, "Cast Fusion Elemental", true);
```

Comment:
oh interesting, I usually write the description to state the expected behavior, and you're writing it as the error message. Confused me but I guess both styles are valid!